rpc/ethereum/backend/backend.go

```diff
@@ -131,7 +131,7 @@ func (e *EVMBackend) BlockNumber() (hexutil.Uint64, error) {
 		return 0, fmt.Errorf("unexpected '%s' gRPC header length; got %d, expected: %d", grpctypes.GRPCBlockHeightHeader, headerLen, 1)
 	}
 
-	height, err := strconv.ParseUint(blockHeightHeader[0], 10, 64)
+	height, err := strconv.ParseUint(blockHeightHeader[0], 10, 63)
```

Comment:
See https://github.com/cosmos/gosec/tree/master/rules/sdk#strconv-unsigned-integers-cast-to-signed-integers-overflow

Reply:
That's odd, thanks for checking. Let's just take note of this for now and move on.